static/js/panels/ProfileForm.tsx

```diff
@@ -75,6 +132,64 @@ const ProfileForm: FC = () => {
                 }
                 stacked
               />
+              <Accordion
+                titleElement="h3"
```

Comment:
I think h3 is a little too big here. Maybe try h4 or h5? Or one of the small text utility classes.